README.ca.md

```diff
@@ -0,0 +1,16 @@
+<img width="150" height="150" align="left" style="float: left; margin: 0 10px 0 0;" alt="MS-DOS logo" src="https://github.com/Microsoft/MS-DOS/blob/master/msdos-logo.png">   
+
+# Codi font de MS-DOS v1.25 i v2.0
+Aquest repositori conté el codi font i binaris compilats originals per a MS-DOS v1.25 i per a MS-DOS v2.0.
+
+Aquests són els mateixos fitxers [originalment compartits al Museu Històric dels Computadors el 25 de Març del 2014]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) i estan sent (re)publicats en aquest repositori per a que siguin més senzills de trobar, de referenciar en escrits i treballs externs, i permetre la exploració i experimentació per a aquells interessats en Sistemes Operatius de PC primerencs.  
+
+# Llicència
+Tots els fitxers dintre d'aquest repositori estan alliberats sota la [Llicència MIT (OSI)]( https://en.wikipedia.org/wiki/MIT_License) segons el [fitxer de LLICÈNCIA](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) emmagatzemat en l'arrel d'aquest repositori.
+
+# Contribueix!
+Els fitxers font en aquest repositori son per referència històrica i es mantindran estàtics, així que, si us plau, no envieu Pull Requests suggerint qualsevol modificació als arxius font, però sé lliure de duplicar aquest repositori i experimentar 😊.  
+
+Però, si dessitges enviar contingut adicional que no sigui d'origen o modificacions a arxius que no siguin de font (per exemple, aquest arxiu README), si us plau envia un PR, i ho revisarem i considerarem.
```

Comment:
Oh wow, you're right! Sorry about that. :slightly_frowning_face:   I corrected it and made another pull request, since this PR has already been merged.